services/ui/ws/window_tree.h

```diff
@@ -582,9 +582,9 @@ class WindowTree : public mojom::WindowTree,
   void GetWindowManagerClient(
       mojo::AssociatedInterfaceRequest<mojom::WindowManagerClient> internal)
       override;
-  void GetWindowTreeHostFactory(
-      mojo::AssociatedInterfaceRequest<mojom::WindowTreeHostFactory> request)
-      override;
+  void GetExternalWindowTreeHostFactory(
```

Comment:
please do git cl format on this file as well

Reply:
this is actually the outcome of git cl format :/

Reply:
ok :+1: